website/docs/r/workitemquery_permissions.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_workitemquery_permissions"
+description: |-
+  Manages permissions for Work Item Queries
+---
+
+# azuredevops_git_permissions
+
+Manages permissions for Work Item Queries. 
+
+~> **Note** Permissions can be assigned to group principals and not to single user principals.
+
+## Permission levels
+
+Permission for Work Item Queries within Azure DevOps can be applied on two different levels.
+Those levels are reflected by specifying (or omitting) values for the arguments `project_id` and `path`.
+
+### Project level
+
+Permissions for all Work Item Queries inside a project (existing or newly created ones) are specified, if only the argument `project_id` has a value.
+
+#### Example usage
+
+```hcl
+resource "azuredevops_workitemquery_permissions" "project-wiq-root-permissions" {
+  project_id  = azuredevops_project.project.id  
+  principal   = data.azuredevops_group.project-readers.id
+  permissions = {
+    CreateRepository = "Deny"
+    DeleteRepository = "Deny"
+    RenameRepository = "NotSet"
+  }
+}
+```
+
+### Shared Queries folder level
+
+Permissions for a specific folder inside Shared Queries are specified if the arguments `project_id` and `path` are set.
+
+~> **Note** To set permissions for the Shared Queries folder itself use `/` as path value
+
+#### Example usage
+
+```hcl
+resource "azuredevops_workitemquery_permissions" "wiq-folder-permissions" {
+  project_id = azuredevops_project.project.id
+  path = "/Team"
+  principal   = data.azuredevops_group.project-readers.id
+  permissions = {
+    Contribute = "Allow"
+    Delete = "Deny"
+    Read = "NotSet"
+  }
+}
+```
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  project_name       = "Test Project"
+  description        = "Test Project Description"
+  visibility         = "private"
+  version_control    = "Git"
+  work_item_template = "Agile"
+}
+
+data "azuredevops_group" "project-readers" {
+  project_id = azuredevops_project.project.id
+  name       = "Readers"
+}
+
+data "azuredevops_group" "project-contributors" {
+  project_id = azuredevops_project.project.id
+  name       = "Contributors"
+}
+
+data "azuredevops_group" "project-administrators" {
+  project_id = azuredevops_project.project.id
+  name       = "Project administrators"
```

Comment:
`Project administrators`  should  be `Project Administrators`  in most scenario

Reply:
Updated the sample